experimenter/experimenter/settings.py

```diff
@@ -60,7 +58,7 @@
 ALLOWED_HOSTS = [HOSTNAME]
 
 if DEBUG:
-    ALLOWED_HOSTS += ["localhost", "nginx"]  # pragma: no cover
+    ALLOWED_HOSTS += ["localhost", "nginx", "0.0.0.0"]  # pragma: no cover
```

Comment:
@jaredlockhart after changing app to experimenter, I am not sure what I am missing but localhost is not working, I have to explicitly add 0.0.0.0, do you know why?